gqlalchemy/query_builder.py

```diff
@@ -609,7 +640,9 @@ def return_(self, results: Optional[Dict[str, str]] = {}) -> "DeclarativeBase":
 
         return self
 
-    def order_by(self, properties: str) -> "DeclarativeBase":
+    def order_by(
+        self, properties: Union[str, Tuple[str, Order], List[Union[str, Tuple[str, Order]]]]
+    ) -> "DeclarativeBase":
```

Comment:
language question, why do we need qoutes around DeclarativeBase?

Reply:
https://peps.python.org/pep-0484/#forward-references